client/src/pages/login/applicantDash/applicant.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './applicant.css';
+
+function Applicant() {
+  return (
+  	<div className="App-applicant" id="Applicant">   {/* This code basically puts "Applicant Dash"*/}
+	  	<div className="container">					 {/* in the middle of the page */}
+		  	<div className="applicant">
+		  	<h1>Applicant Dash</h1>   {/* Big word "Applicant Dash" */}
```

Comment:
Two smaller things here that come up in a few different files:

1) I don't know how much the comments here are aiding in code readability. I think any dev will understand the point of a ```h1``` tag.
2) Your ```id="Applicant"``` and ```classname="applicant"``` are awfully similar. Perhaps one or both could be more specific to avoid the almost-identical names?

Reply:
1. Ah, right haha, my bad. 
2. Changed/removed the "id"s to make it less repetitive. Also clarified vague css classifications.